.github/workflows/ci.yml

```
@@ -126,6 +126,44 @@ jobs:
           ninja
           ../scripts/ensure-snmalloc.sh samples/apps/basic/basic
 
+  check-doc:
+    name: "Build docs successfully"
+    runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
+    container:
+      image: mcr.microsoft.com/azurelinux/base/core:3.0
+      options: --user root --publish-all --cap-add NET_ADMIN --cap-add NET_RAW --cap-add SYS_PTRACE
```

```suggestion
      options: --user root
```

I think the user can probably go too, but the rest definitely can.